connaisseur/tests/integration/integration-test.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+set -euo pipefail
+
+envsubst <connaisseur/tests/integration/values.yaml >helm/values.yaml
+
+make install || { echo "Failed to install Connaisseur"; exit 1; }
```

Comment:
I noticed that on my machine `make install` can get stuck and I am not sure whether it times out. That would potentially make the ci freeze here, right?

Reply:
If it times out, it would "freeze" for the 10min timeout then die
Not sure what it would be if not a timeout. Why are you not sure whether it's a timeout on your end? But yes, if `make install` produced an endless loop, then this would be an endless loop and "freeze" CI